questions/understand-git-commands-questions.md

```diff
@@ -5,20 +5,59 @@ Complete the questions below, by editing this file. Please commit often, to your
 You may reference the videos, the cheatsheet or even google to complete these questions. You can even use this project to play with these git commands.
 
 1. Does a commit connect to the remote?
-1. Does a push connect to the remote?
-1. What is a git repo even?
-1. How do you stage a file, and what does it mean?
-1. How do you unstage a file?
-1. What is the output of `git log`?
-1. How is a git repo initialised?
-1. What information will `git status` give you?
-1. What does a git fetch do?
-1. A `git pull` can basically be broken down into these git commands:
-1. What is a rebase?
-1. How do you view information about the remote config of a git repo?
-1. What commands will create a branch called `qa` from `master`
-1. Lets say you've made changes which you want to discard, and you have not yet commited them anyway, what command can you use?
-1. What is the difference between `git merge` and `git merge <branch>`
-1. What is the git command to change a branch's name locally and remotely? 
-1. If you commit a file, is it commited to your current branch or to master?
-1. If you commited a change, but it was for the wrong branch, how can you move the commit?
+No, it does not it adds to the bracnh you working on.
+
+2. Does a push connect to the remote?
+Yes it does as you pushing your changes to some where.
+
+3. What is a git repo even?
+It is a version control system that contains the history of all files worked on.
+
+4. How do you stage a file, and what does it mean?
+When you execute the command git add you are stagging the file getting it ready to be added.
+
+5. How do you unstage a file?
+You use the git reset file name.
+
+6. What is the output of `git log`?
+It gives you the history of what took place on your branch or project.
```

Comment:
Correct, but not just your branch, it will actually show you where you branched off...